examples/multiple-choice/index.html

```diff
@@ -41,8 +41,8 @@
       [
         {
           question: 'What framework should we start with?',
-          answers: ['React', 'Ember.js', 'Vue'],
-          answer: 1
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'jQuery'],
```

Comment:
arguing the semantics of what is/isn't a framework points to the misleading nature of the original question... it assumes the options should be limited to true frameworks, as if other possibilities have already been eliminated. the real question that probably matters is "what popular tool for developing the client side application should we use?"